### PR TITLE
add notebook and sample data

### PR DESCRIPTION
**Pull request by MuhammadAgf**

### CODE REVIEW

**Comment by MuhammadAgf** [comment]
@carash plz review thx

File: data/example/character/fischl.yml [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+element: electro
+weapon_type: Bow
+base_stats:
+- Lv: '1'
+  Base HP: '770'
+  Base ATK: '20'
+  Base DEF: '50'
+  ATK%: '0'
+  CRIT Rate: '5'
+  CRIT DMG: '50'
```
Comment (carash) [inline data/example/character/fischl.yml]: IMO storing stats should be stored in list with name and value
```suggestion
  Stats:
  - Name: Base HP
    Value : 770
  - Name: Base ATK
    Value : 20
  - Name: Base DEF
    Value : 50
  - Name: ATK%
    Value : 0
  - Name: Crit Rate
    Value : 5
  - Name: Crit Damage
    Value : 50
```

File: data/example/character/fischl.yml [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+- Lv: '90'
+  Base HP: '9189'
+  Base ATK: '244'
+  Base DEF: '594'
+  ATK%: '24'
+  CRIT Rate: '5'
+  CRIT DMG: '50'
+normal_attack:
+  Lv1:
+    1-Hit DMG: 44.12%
```
Comment (carash) [inline data/example/character/fischl.yml:203]: imo % should not be stored

File: data/example/character/fischl.yml [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+base_stats:
+- Lv: '1'
+  Base HP: '770'
+  Base ATK: '20'
+  Base DEF: '50'
+  ATK%: '0'
+  CRIT Rate: '5'
+  CRIT DMG: '50'
+- Lv: '20'
+  Base HP: '1979'
```
Comment (carash) [inline data/example/character/fischl.yml]: pls store as number

**Review by carash — CHANGES_REQUESTED** [review]
Yey being a critic is fun

File: data/example/character/fischl.yml (hunk 1 above)
Comment (MuhammadAgf) [reply data/example/character/fischl.yml]: would you mind providing example for this?

File: data/example/character/fischl.yml (hunk 2 above)
Comment (MuhammadAgf) [reply data/example/character/fischl.yml:203]: hmm, how to differentiate percentage and raw number then?

File: data/example/character/fischl.yml (hunk 3 above)
Comment (MuhammadAgf) [reply data/example/character/fischl.yml]: males ✌️ 


gadeng, nnt w edit

File: data/example/character/fischl.yml (hunk 2 above)
Comment (carash) [reply data/example/character/fischl.yml:203]: ah right, ok then

File: data/example/character/fischl.yml (hunk 1 above)
Comment (MuhammadAgf) [reply data/example/character/fischl.yml]: ah ya w kobam lol, okok

Comment (carash) [reply data/example/character/fischl.yml]: image

File: data/example/character/fischl.yml (hunk 3 above)
Comment (MuhammadAgf) [reply data/example/character/fischl.yml]: updated om

File: data/example/character/fischl.yml (hunk 1 above)
Comment (MuhammadAgf) [reply data/example/character/fischl.yml]: updated om

**Review by carash — APPROVED** [review]

**Pull request merged**